### PR TITLE
Change version to 3.2

### PR DESCRIPTION
**Pull request by feiling**
@zhili1208 @emgarten @deepakaravindr @yishaigalatzer

### CODE REVIEW

File: src/NuGet.Client/project.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
         "warningsAsErrors": true
     },
     "dependencies": {
-        "NuGet.Versioning": "3.1.2-*",
-        "NuGet.Packaging": "3.1.2-*",
-        "NuGet.Repositories": "3.1.2-*",
-        "NuGet.RuntimeModel": "3.1.2-*",
-        "NuGet.ContentModel": "3.1.2-*",
-        "NuGet.Common": { "version": "3.1.2-*", "type": "build" }
+        "NuGet.Versioning": "3.2-*",
```
Comment (yishaigalatzer) [inline src/NuGet.Client/project.json:7]: @davidfowl changing versions is painful. in msbuild we can use a macro for these values. Is there a mechanism like that in project.json ?

File: src/NuGet.Protocol.ApiApps/project.json [hunk 2]
```diff
@@ -11,8 +11,8 @@
     },
     "dependencies": {
         "Newtonsoft.Json": "6.0.6",
```
Comment (yishaigalatzer) [inline src/NuGet.Protocol.ApiApps/project.json:13]: Why are we changing API Apps?

File: src/NuGet.Packaging.Build/project.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ﻿{
-    "version": "3.1.2-*",
+    "version": "3.2-*",
     "preprocess": "../../compiler/preprocess/*.cs",
     "dependencies": {
-        "NuGet.Versioning": "3.1.2-*",
-        "NuGet.Packaging": "3.1.2-*",
+        "NuGet.Versioning": "3.2-*",
+        "NuGet.Packaging": "3.2-*",
         "Newtonsoft.Json": "6.0.6",
```
Comment (yishaigalatzer) [inline src/NuGet.Packaging.Build/project.json:7]: In Visual Studio we use 6.0.4 (must) why are we compiling against 6.0.6 ?

File: src/NuGet.Protocol.Core.v2/project.json [hunk 4]
```diff
@@ -10,11 +10,11 @@
         "warningsAsErrors": true
     },
     "dependencies": {
-        "NuGet.Configuration": "3.1.2-*",
-        "NuGet.Packaging": "3.1.2-*",
-        "NuGet.Protocol.Core.Types": "3.1.2-*",
+        "NuGet.Configuration": "3.2-*",
+        "NuGet.Packaging": "3.2-*",
+        "NuGet.Protocol.Core.Types": "3.2-*",
         "NuGet.Core": "2.8.6",
```
Comment (yishaigalatzer) [inline src/NuGet.Protocol.Core.v2/project.json:16]: This is supposed to change to 2.8.8

**Comment by yishaigalatzer** [comment]
Do we have any strings in the code as well for versions?

File: src/NuGet.Protocol.ApiApps/project.json (hunk 2 above)
Comment (feiling) [inline src/NuGet.Protocol.ApiApps/project.json:13]: I think we want all packages to have the same version.

File: src/NuGet.Protocol.Core.v2/project.json (hunk 4 above)
Comment (feiling) [inline src/NuGet.Protocol.Core.v2/project.json:16]: That will be in another pull request.

File: src/NuGet.Packaging.Build/project.json (hunk 3 above)
Comment (feiling) [inline src/NuGet.Packaging.Build/project.json:7]: Not sure about that. @emgarten @deepakaravindr, does any one of you know the answer?

**Comment by feiling** [comment]
No, there are no strings containing "3.1.2" in the solution.

**Comment by emgarten** [comment]
Use 3.2.0 instead of 3.2 :watch:

**Comment by feiling** [comment]
version changed to "3.2.0".
